src/api/common/HttpClient.ts

```diff
@@ -70,7 +71,25 @@ export default class HttpClient {
 
     return response;
   }
+
+  async getPingResponse(
+    relayUrl: string,
+    verifier?: string
+): Promise<PingResponse> {
+    const verifierSuffix = verifier == null ? '' : '?verifier=' + verifier;
+    const pingResponse = await this._httpWrapper.sendPromise<PingResponse>(
+        relayUrl + '/getaddr' + verifierSuffix
+    );
+    if (pingResponse == null) {
+        throw new Error('Relay responded without a body');
+    }
+    log.info(`pingResponse: ${JSON.stringify(pingResponse)}`);
+
+    return pingResponse;
+  }
 }
 
+export const defaultHttpClient = new HttpClient(new HttpWrapper());
```

Comment:
```suggestion
export const getDefaultHttpClient = (
    opts: AxiosRequestConfig = {},
    loglLevel: LogLevelDesc = 'error'
  ) => new HttpClient(new HttpWrapper(opts, logLevel));
```

though I'd also consider using a static method allowing the use of singleton.

Reply:
I'll use the approach you wrote. I have never be a pro singleton pattern in JS.